### ISSUE TITLE
Build pytest<7 for pypy3.8/3.9 (and eventually python 3.11)

### ISSUE THREAD

**Issue by mattip** [conda-forge/pytest-feedstock#149]
### Comment:

The ipython feedstock and maybe others requires pytest<7 because of https://github.com/pytest-dev/pytest/issues/9765. See conda-forge/ipython-feedstock#174. What would be the easiest way to create a PR with the pypy3.8/3.9 migration from PR #146 and the pre-7 commit? I think a merge request will need a HEAD of some branch.

**Comment by nicoddemus** [comment]
I think you are right, we would need a `6.x` branch or something like that.

Would you like to open a PR changing the version and targeting `master`? Then I can push a new branch and change the target to a new `6.x` branch.

**Comment by mattip** [comment]
Done in https://github.com/mattip/pytest-feedstock/commits/v6.x
```
git checkout 0ec6592f6c834848b124f3b8b5ffece716c6f4b1     # the last commit before the 7.0 PR
git checkout -b v6.x
git cherry-pick 464d9aa0c1dde1667d7aa08a9e95d68e624fddc9  # the PyPy3.8/3.9 migration
conda smithy rerender
git commit -m "MNT: Re-rendered with conda-build 3.21.8, conda-smithy 3.21.1, and conda-forge-pinning 2022.08.23.12.33.00"
git push mattip
```

So if I understand correctly an owner could do
```
git add remote mattip https://github.com/mattip/pytest-feedstock
git fetch mattip v6.x
git push origin   # <- this is the admin bit that creates a new branch, right?
```

then a PR of my branch against origin (this repo) would target the new branch.

**Comment by nicoddemus** [comment]
Indeed that's what is in the docs, but I feel it would be better for you to open a PR against the `6.x` branch (which I just pushed), so we can run it over CI first. 👍

**Comment by mattip** [comment]
Closed by #150
